go/host/enclave/guardian.go

```diff
@@ -74,7 +76,7 @@ type Guardian struct {
 	enclaveID        *common.EnclaveID
 }
 
-func NewGuardian(cfg *config.HostConfig, hostData host.Identity, serviceLocator guardianServiceLocator, enclaveClient common.Enclave, db *db.DB, interrupter *stopcontrol.StopControl, logger gethlog.Logger) *Guardian {
+func NewGuardian(cfg *config.HostConfig, hostData host.Identity, serviceLocator guardianServiceLocator, enclaveClient common.Enclave, db *sql.DB, interrupter *stopcontrol.StopControl, logger gethlog.Logger) *Guardian {
```

Comment:
Extremely minor niggle and maybe Tudor commented about this already, but would be nice if the dependency didn't mention sql.

Guardian doesn't care what database is getting used, even if it's likely to always be a sql db imo it would be slightly cleaner to have a generic db interface.

Reply:
Yeah this has been replaced with a storage interface similar to the enclave and all interactions go through that instead 👍🏻